packager/packager.js

```diff
@@ -35,6 +35,14 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'protocol',
```

Comment:
The protocol will almost always be http (though https enables HTTP/2) but separating it from the hostname makes it easier to configure only the hostname and leave the protocol/port alone, which is a common thing to do.